packages/@webex/plugin-cc/test/unit/spec/services/task/index.ts

```diff
@@ -0,0 +1,109 @@
+import 'jsdom-global/register';
+import {LocalMicrophoneStream} from '@webex/calling';
+import {LoginOption} from '../../../../../src/types';
+import { CC_FILE } from '../../../../../src/constants';
+import Task from '../../../../../src/services/task';
+import * as Utils from '../../../../../src/services/core/Utils';
+
+jest.mock('@webex/calling');
+
+describe('Task', () => {
+  let task;
+  let contactMock;
+  let taskDataMock;
+  let webCallingServiceMock;
+  let getErrorDetailsSpy;
+  const taskId = 'taskId123';
+
+  beforeEach(() => {
+    contactMock = {
+      accept: jest.fn().mockResolvedValue({}),
+    };
+
+    webCallingServiceMock = {
+      loginOption: LoginOption.BROWSER,
+      answerCall: jest.fn(),
+      declineCall: jest.fn()
+    };
+
+      // Mock task data
+      taskDataMock = {};
+
+      // Create an instance of Task
+      task = new Task(contactMock, webCallingServiceMock, taskDataMock);
+
+      // Mock navigator.mediaDevices
+      global.navigator.mediaDevices = {
+        getUserMedia: jest.fn(() =>
+          Promise.resolve({
+            getAudioTracks: jest.fn().mockReturnValue([''])
+          })
+        ),
+      };
+      
+      // Mock MediaStream (if needed)
+      global.MediaStream = jest.fn().mockImplementation((tracks) => {
+        return {
+          getTracks: jest.fn(() => tracks),
+        };
+      });
+
+      getErrorDetailsSpy = jest.spyOn(Utils, 'getErrorDetails')
+  });
+
+
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('should accept a task and answer call when using BROWSER login option', async () => {
+    await task.accept(taskId);
+
+    expect(navigator.mediaDevices.getUserMedia).toHaveBeenCalledWith({ audio: true });
```

Comment:
shouldn't we mock this function to return the object we want and then check that in the next line?

Reply:
This is addressed